common/build/eslint-config-fluid/package.json

```diff
@@ -26,8 +26,5 @@
   "devDependencies": {
     "eslint": ">=8.6.0",
     "typescript": ">=4.5.3"
-  },
```

Comment:
maybe not entirely related, but any idea if we are stuck publishing these on npmjs along with our other packages in order to use them how we do?

Reply:
One alternative is to have the build-tools be built in the install step, and have the package assume it's relative path in the repo, or symlink it ourselves.

The other alternative is that if our repo becomes one mono-repo, then, building the build-tools can be the first build step.  It will add to the build time, and strongly tie the tool to this one mono-repo.

Not sure there is a best choice here.

Reply:
My preference would be to symlink the build tool as part of the workspace by default, but retain the option to publish a version and pin to it in the future if we need to do a staged upgrade of the tools.